specs/darwin/keychain_items.table

```diff
@@ -7,6 +7,6 @@ schema([
     Column("created", TEXT, "Data item was created"),
     Column("modified", TEXT, "Date of last modification"),
     Column("type", TEXT, "Keychain item type (class)"),
-    Column("path", TEXT, "Path to keychain containing item", index=True),
+    Column("path", TEXT, "Path to keychain containing item", additional=True),
```

Comment:
I _do_ see special handling on the this path. It really _ought_ be an index, though it's non-unique. I wonder if it makes sense to set the `label` and `type` as additional and leave this as index.

Reply:
I didn't see any specific set of columns here that together can be used as a primary key.